pkg/scheduler/internal/queue/scheduling_queue.go

```diff
@@ -155,6 +156,8 @@ type PriorityQueue struct {
 	stop  chan struct{}
 	clock clock.Clock
 
+	qLock sync.RWMutex
```

Comment:
How about rename the nominator lock to minimize the change.

Reply:
Nominator lock is named to `nominatorLock`, but the `lock` was renamed to `qLock` according to the [comment](https://github.com/kubernetes/kubernetes/pull/125820#discussion_r1660773953)

Reply:
em.. the name is ok to me. We don't have other locks named `lock`, but this is personal biases.

Reply:
If you mean to turn back the `nominatorLock` name to `lock`, I think it might be mistaken in the future, as the `nominator` is embedded into the `PriorityQueue`

Reply:
Turned back the main lock name to `lock`. Renamed nominator's lock to `nLock`